students/jeonghyeon/users/vaildation.py

```diff
@@ -0,0 +1,15 @@
+import re
+from django.forms import ValidationError
+
+
+REGEX_PASSWORD ='^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$'
+REGEX_EMAIL = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+
+
+def Vaildation_email(email):
+    if not re.match(REGEX_EMAIL, email):
+        raise ValidationError('Check_email')
```

Comment:
에러를 리턴해줄거면 보다 명확한 메시지가 필요할 것 같습니다.
```suggestion
        raise ValidationError('INVALID_EMAIL')
```

Reply:
명확한 에러메시지로 변경 완료